fineract-provider/build.gradle

```diff
@@ -125,6 +125,10 @@ rat {
     '**/native/**',
     '**/wrapper/**',
 	'**/build/**',
+    // docker
+    '**/Dockerfile/**',
+    '**/docker-compose.yml/**',
+    '**/initdb/**',
```

Comment:
it just occured to me that it would probably be better to add the ASF required license headers instead of suprressing this here like this... at least for YAML files (like docker-compose.yml) it's possible, see the https://github.com/apache/fineract/blob/develop/.travis.yml, and for 01-databases.sql it's possible as well, see e.g. the https://github.com/apache/fineract/blob/develop/fineract-db/mifospltaform-tenants-first-time-install.sql et all. Not 100% sure re. the Dockerfile, but just try adding a license header to it, and see if rat (the license check tool, which you can run locally via `./gradlew licenseMain licenseTest licenseIntegrationTest`) is happy, without having to add these ignores?

Reply:
Thanks for your feedback. I updated all 3 of those files and added the license header. RAT seems fine with it. As far as the changes to build.gradle, if I leave the DB host as localhost, the docker images crashes on startup when Fineract runs the Flyway migration:

FlywayException: Unable to obtain Jdbc connection from DataSource; for Tenant DB URL: jdbc:mysql:thin://localhost:3306/mifostenant-default, username: root

I can dig into it a bit more, but appears that some part of the gradle build configuration is carried into the war file...

Reply:
Flyway will pick the tenant connection info from the mifosplatform-tenant DB and use that to try and connect to the different tenants. By default, the mysql host value is localhost. This is what causes the problem in the container at startup.
What you can do is to copy and modify the list-db SQL scripts to change the mysql host to the dockerhost of mysql. Its a dirty fix.

Reply:
Duh! OK.. I still don't entirely get what's going on here, but don't want to hold this up any further, and do trust you (we can always get back to fully resolving this mystery in [FINERACT-765](https://issues.apache.org/jira/browse/FINERACT-765)). Thanks for your patience with me re. this point.

Reply:
see #613